### PR TITLE
initial push landing page

### PR DESCRIPTION
**Pull request by KenMain5**
please review. I made comments to explain the functionalities in some of the files.

### CODE REVIEW

File: src/components/landingpage/cards/Cards.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+function Cards({ icon, title, text }) {
```
Comment (clee4037) [inline src/components/landingpage/cards/Cards.jsx:1]: Implementing redux will make passing props easier. You can use my reservation slice files as an example.

File: src/assets/landingpage/carouselPhoto1.jpg
Comment (clee4037) [inline src/assets/landingpage/carouselPhoto1.jpg:1]: Nice photo selection

**Review by clee4037 — APPROVED** [review]
Styling and organization looks good. I recommend implementing redux

**Review by kssampson — APPROVED** [review]
Nice photo choices and the carousel at the bottom will look nice.

**Pull request merged**